src/rime/dict/level_db.cc

```diff
@@ -54,7 +54,7 @@ struct LevelDbWrapper {
   leveldb::Status Open(const path& file_path, bool readonly) {
     leveldb::Options options;
     options.create_if_missing = !readonly;
-    return leveldb::DB::Open(options, file_path.string(), &ptr);
+    return leveldb::DB::Open(options, file_path.u8string(), &ptr);
```

Comment:
passing `u8string()` is incorrect.
leveldb uses Win32 `*A` API.
https://github.com/google/leveldb/blob/068d5ee1a3ac40dabd00d211d5013af44be55bea/util/env_windows.cc

Reply:
how stupid leveldb is! it's been 4202...maybe let's rocksdb with utf8 supported all platforms.

Reply:
> passing `u8string()` is incorrect. leveldb uses Win32 `*A` API. https://github.com/google/leveldb/blob/068d5ee1a3ac40dabd00d211d5013af44be55bea/util/env_windows.cc

I wrote a patch to make UTF-8 string acceptable to leveldb, maybe a solution